images/gitness/tests/01-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+# Set the name of the image if needed
+if [[ "${IMAGE_NAME}" == "" ]]; then
+    echo "Must set IMAGE_NAME environment variable. Exiting."
+    exit 1
+fi
+
+uid="gitness-$FREE_PORT"
+
+docker run -d \
+  -p $FREE_PORT:3000 \
+  -v /var/run/docker.sock:/var/run/docker.sock \
```

Comment:
Hmm, why does it need access to the daemon?

Reply:
It's a full-fledged CI server that does fun DIND stuff.